AzureCalling/Controllers/LobbyViewController.swift

```diff
@@ -243,6 +245,19 @@ class LobbyViewController: UIViewController, UITextFieldDelegate {
         startCallButton.isEnabled = isDisplayNameValid && isPermissionValid
     }
 
+    // update the audio device icon with current selection
+    private func updateAudioDevice() {
+        let currentAudioDeviceType = AudioSessionManager.getCurrentAudioDeviceType() // Get the current audio selection
+        // set icon based on audio type
+        var deviceIcon: UIImage
+        switch currentAudioDeviceType {
+        case .receiver:
+            deviceIcon = UIImage(named: "ic_fluent_speaker_2_28_regular")!
+        case .speaker:
+            deviceIcon = UIImage(named: "ic_fluent_speaker_2_28_filled")!
+        }
```

Comment:
Similar code with different naming for `deviceIcon` and `image` with CallViewController. updateAudioDevice

Can extract a function to return the UIImage